.github/workflows/refreeze-dockerfile-requirements-txt.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
               --volume=$PWD:/opt/${{ matrix.image }} \
               --workdir=/opt/${{ matrix.image }} \
               --user=root \
-              python:3.11-slim-bullseye \
+              python:3.13-slim-bullseye \
```

Comment:
This should match the base image used in Dockerfiles you'll find for the heln chart, bump thosr as well or omit this bump.

Reply:
I updated the dockerfiles now :)